src/shared/Microsoft.Git.CredentialManager/Settings.cs

```diff
@@ -163,6 +183,13 @@ public class ProxyConfiguration
         public ICollection<string> BypassHosts { get; }
     }
 
+    public enum TlsBackend
+    {
+        Schannel,
+        OpenSsl,
+        Other,
+    }
```

Comment:
I wonder if we include a `TlsBackend.Default = 0` option, or just put `TlsBackend.OpenSsl` at position `0` (and therefore it becomes the `default(TlsBackend)` value)?

@dscho is `http.sslBackend` ever likely to change the default from `openssl`? Is there a way to ask Git what the default value is for a configuration option when unset?

Reply:
`http.sslBackend`'s default is likely to remain unchanged, barring any compelling reason, of course.